public/css/home.css

```diff
@@ -4,3 +4,18 @@
   padding: 36px;
   color: black;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.navbar {
+  transition-timing-function: ease;
+  transition: 0.5s;
+}
+
+.cover {
+  background-image: url('../img/home/cover-bg.png');
+  background-size: cover;
+  background-position: bottom left;
```

Comment:
are there any bootstrap classes for these 2 rules? if not, ignore this comment. This is fine

Reply:
No Bootstrap for that but i can make it shorthand, which i have updated.